viaget.go

```diff
@@ -0,0 +1,297 @@
+package kramer
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+// User status constants
+const (
+	Inactive = "0"
+	Active   = "1"
+	Waiting  = "2"
+)
+
+func (v *Via) GetVolume(ctx context.Context) (int, error) {
+	var cmd command
+	cmd.Command = "Vol"
+	cmd.Param1 = "Get"
+
+	v.Infof("Sending command to get VIA Volume to %s", v.Address)
+	// Note: Volume Get command in VIA API doesn't have any error handling so it only returns Vol|Get|XX or nothing
+	// I am still checking for errors just in case something else fails during execution
+	vollevel, err := v.sendCommand(ctx, cmd)
+	if err != nil {
```

Comment:
I'm a little confused with this part. The comment states we are still checking for errors just in case, but we don't do anything if there happens to be an error. 

If that's the correct logic flow, then we are better off ignoring the error (ie `vollevel, _ := v.sendCommand(ctx, cmd)`. If we should do something in the rare case of an error, then let's do something with that error like return it.